packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -153,6 +165,36 @@ contract Deploy is Deployer {
         deployProtocolVersions();
     }
 
+    // @notice Gets the address of the SafeProxyFactory and Safe singleton for use in deploying a new GnosisSafe.
+    function _getSafeFactory() internal returns (SafeProxyFactory safeProxyFactory_, Safe safeSingleton_) {
+        if (_isDevnet() == true) {
```

Comment:
An alternative here would be to check if the code exists and deploy if not but ok with modularizing the is devnet check too

Reply:
i feel like we should do the no code check because then otherwise live networks without safe infra will break

Reply:
OK, adding that.